src/logger.c

```diff
@@ -364,16 +364,16 @@ void log_init(char *log_name, char *pot_name, char *session)
 	show_admins = cfg_get_bool(SECTION_OPTIONS, NULL, "MarkAdminCracks", 0);
 
 	if (isatty(fileno(stdout))) {
-		admin_start = parse_esc(cfg_get_param(SECTION_OPTIONS, NULL,
+		admin_start = parse_esc((char*)cfg_get_param(SECTION_OPTIONS, NULL,
 		                                      "MarkAdminStart"));
-		admin_end = parse_esc(cfg_get_param(SECTION_OPTIONS, NULL,
+		admin_end = parse_esc((char*)cfg_get_param(SECTION_OPTIONS, NULL,
 		                                    "MarkAdminEnd"));
-		terminal_reset = parse_esc(cfg_get_param(SECTION_OPTIONS, NULL,
+		terminal_reset = parse_esc((char*)cfg_get_param(SECTION_OPTIONS, NULL,
 		                                         "TerminalReset"));
 	} else
 		admin_start = admin_end = terminal_reset = "";
 
-	admin_string = parse_esc(cfg_get_param(SECTION_OPTIONS, NULL,
+	admin_string = parse_esc((char*)cfg_get_param(SECTION_OPTIONS, NULL,
 	                                       "MarkAdminString"));
```

Comment:
I'm fixing all of the above in a separate commit, changing parse_esc to not alter string in-place.

Reply:
Scrap "separate commit" - I'm squashing it, just later.